### PR TITLE
Add links to team prod and qa deployments

### LINKED ISSUE #29: Set up team prod and qa deployment
- [x] Setup team prod deployment at https://organic.dokku-05.cs.ucsb.edu/
- [x] Setup team qa deployment at https://organic-qa.dokku-05.cs.ucsb.edu/

### PR DESCRIPTION
**Pull request by Joseph-Foster**
Update the README with links to prod and qa deployments
Closes #29

### CODE REVIEW

**Review by ErwanDev — DISMISSED** [review]
LGTM, deployment links are correct and running.

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
```
Comment (andrewpengucsb) [inline README.md:4]: While you're at it, you should change the header of the readme as well to match the repo

**Review by andrewpengucsb — DISMISSED** [review]

**Review by chauAlex — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**